src/CmsBundle/Entity/User.php

```diff
@@ -56,8 +56,20 @@ class User extends FOSUser implements TwoFactorInterface
     protected $email;
 
     /**
-     * @var string
+     * @Assert\Length(
+     *      min = 8,
+     *      max = 50,
+     *      minMessage = "Your password must be at least {{ limit }} characters long",
+     *      maxMessage = "Your password cannot be longer than {{ limit }} characters"
+     * )
+     *
+     * @Assert\Regex(
+     *     pattern="/^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)[a-zA-Z\d]+$/",
```

Comment:
This does not match `Afgjhdf5kgklfBhgdkhkh}df`

Reply:
fixed